05_syntax_lab/01.py

```diff
@@ -1,5 +1,19 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+
+write program that gets from user 10 numbers and prints the biggest
+
+
+"""
+
+
+max =0
+
+for i in range (1,11):
+
+	num = int(raw_input())

```

Comment:
raw_input() can take an argument and then uses it as a prompt value, so better to use: `num = int(raw_input("Type your number: "))`